src/main/java/org/killbill/billing/plugin/adyen/api/ExpiredPaymentPolicy.java

```diff
@@ -89,4 +91,8 @@ private boolean is3ds(final AdyenPaymentTransactionInfoPlugin transaction) {
         final AdyenResponsesRecord adyenResponsesRecord = transaction.getAdyenResponseRecord().get();
         return PaymentServiceProviderResult.REDIRECT_SHOPPER.toString().equals(adyenResponsesRecord.getResultCode());
     }
+
+    private String getPaymentMethod(final PaymentTransactionInfoPlugin transaction) {
+        return PluginProperties.findPluginPropertyValue("paymentMethod", transaction.getProperties());
```

Comment:
I'm not sure, but I think we only have this available once we receive a notification from Adyen (which will should transition the payment).

I remember we talked about adding the `brandCode` parameter to the `adyen_hpp_requests`. Then we can start using this table to both figure out if a payment is HPP and what's the HPP PM used.